lib/Controller/ViewController.php

```diff
@@ -148,4 +149,18 @@ private function getView(string $view): string {
 				return $view;
 		}
 	}
+
+	/**
+	 * @NoCSRFRequired
```

Comment:
no admin required

Reply:
instead or in addition?

Reply:
either. I don't know how client auth works. they might or might not send a CSRF token.

there is also a chance that the URL doesn't work at all for clients. AFAIK they have to be OCS